rotkehlchen/tests/utils/accounting.py

```diff
@@ -86,10 +86,11 @@ def _get_pnl_report_after_processing(
         database: 'DBHandler',
 ) -> tuple[dict[str, Any], list[ProcessedAccountingEvent]]:
     dbpnl = DBAccountingReports(database)
-    report = dbpnl.get_reports(report_id=report_id, with_limit=False)[0][0]
+    report = dbpnl.get_reports(report_id=report_id, with_limit=False, limit=999999999)[0][0]
```

Comment:
is the second time you do something like this. Maybe set a constant in the test folder `BIG_ENOUGH_REPORT_LIMIT: Final = 99999999`. We have a file with such constants if I recall correctly